src/components/layout/v2/grouped-inputs-with-lock.tsx

```diff
@@ -79,7 +79,7 @@ const GroupedInputsWithLock = ({ type, onChange, styles }: Props) => {
               value={(parseInt(styles[name] || styles[type]) || "0").toString()}
               name={name}
               inputProps={{
-                pattern: "[0-9]"
+                pattern: "-[0-9]"
```

Comment:
This pattern matches only texts where first character is - and next one is number, for example -2 but it does not match for -22 or 22 or 2.

You are using parseInt in the value, so it anyway enforces the value to be int, so I would just remove whole pattern

Reply:
True. Resolved.